sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/grpc/FromProtoValueCodecs.java

```diff
@@ -293,86 +326,112 @@ public JsonNode jsonNodeFrom(QueryOuterClass.Value value) {
   protected static final class BooleanCodec extends FromProtoValueCodec {
     @Override
     public Object fromProtoValue(QueryOuterClass.Value value) {
-      return Values.bool(value);
+      return (value.getInnerCase() == QueryOuterClass.Value.InnerCase.NULL)
+          ? null
+          : Values.bool(value);
     }
 
     @Override
     public JsonNode jsonNodeFrom(QueryOuterClass.Value value) {
-      return jsonNodeFactory.booleanNode(Values.bool(value));
+      return (value.getInnerCase() == QueryOuterClass.Value.InnerCase.NULL)
+          ? jsonNodeFactory.nullNode()
+          : jsonNodeFactory.booleanNode(Values.bool(value));
     }
   }
 
   protected static final class UUIDCodec extends FromProtoValueCodec {
     @Override
     public Object fromProtoValue(QueryOuterClass.Value value) {
-      return Values.uuid(value);
+      return (value.getInnerCase() == QueryOuterClass.Value.InnerCase.NULL)
+          ? null
+          : Values.uuid(value);
     }
 
     @Override
     public JsonNode jsonNodeFrom(QueryOuterClass.Value value) {
-      return jsonNodeFactory.pojoNode(Values.uuid(value));
+      return (value.getInnerCase() == QueryOuterClass.Value.InnerCase.NULL)
+          ? jsonNodeFactory.nullNode()
+          : jsonNodeFactory.pojoNode(Values.uuid(value));
     }
   }
 
   protected static final class TimestampCodec extends FromProtoValueCodec {
     @Override
     public Object fromProtoValue(QueryOuterClass.Value value) {
-      return Instant.ofEpochMilli(value.getInt()).toString();
+      return (value.getInnerCase() == QueryOuterClass.Value.InnerCase.NULL)
+          ? null
+          : Instant.ofEpochMilli(value.getInt()).toString();
```

Comment:
@mpenick we've been trying to use the operations on `io.stargate.grpc.Values` where possible but there is no `Values.timestamp() operation. Should there be?

Reply:
It's encoded in the `int` field: https://github.com/stargate/stargate/blob/v1.0.45/grpc-proto/proto/query.proto#L93-L94

Reply:
agree, but what I'm saying is there are operations on `Values`corresponding to other types that can be encoded in `int`:

- `Values.int_()`
- `Values.smallint()`
- `Values.tinyint()`